generate_figures/generalized_mean_deep_sets_no_invariance_N_table.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+import wandb
+from utilities import df_to_latex, get_results_by_tag
+
+output_dir = "./figures"
+
+api = wandb.Api()
+project = "highdimensionaleconlab/symmetry_dynamic_programming"
+
+N = [2, 4, 8, 32, 64, 512, 1024]
+df = pd.DataFrame()
+for n in N:
+    df_n = get_results_by_tag(
+        api, project, f"generalized_mean_no_invariance_N_{n}", get_config=True
+    )
+    # assert df.id.nunique() == whenever we know how many
+    df = pd.concat([df_n, df])
+    df = df[df['retcode'] == 0]
+    
+df_no_invar = pd.DataFrame(df.groupby("model.N")["test_rel_error"].median())
```

Comment:
In general it is better off not to name things with partial names just to save a few characters.  So I would have called this `df_no_invariance` since `invar` isn't clear.   The `df` is a very common abbreviation so it makes things very clear in that case.  The `rel` we use in other places is also pretty common so that is work using (to be consistent).  But otherwise I am a big believer in clarity of funtion names.  Saving a few characters isn't helpful - or necessary with autocomplete in vscode.

When you adapt to the next issue maybe you could rename this to be `df_no_invariance_test_rel_error`   and then for the other one have it in parallel as `df_no_invariance_success` with the separate reduction.  Or something like that.